content/docs/introduction/support.md

```diff
@@ -37,29 +55,127 @@ You can open support tickets in the Neon Console. Look for the **Support** link
 
 You can expect an initial response time of 2 business days, from 6am to 6pm Pacific Standard Time (UTC -8), Monday through Friday, excluding public holidays in the United States. For custom support solutions, please contact [Sales](https://neon.tech/contact-sales).
 
-## Priority support
+### Priority support
 
-Neon's [Scale plan](/docs/introduction/plans#scale) includes **Priority** support.
+Neon's [Scale](/docs/introduction/plans#scale) and [Business](/docs/introduction/plans#business) plans includes **Priority** support.
```

Comment:
Scale is now Standard.

Reply:
@bgrenon ... plans include (not includes) :)

Reply:
Updated on support page and [plans](https://neon-next-git-bgrenon-business-plan-neondatabase.vercel.app/docs/introduction/plans) page.